### ISSUE TITLE
Add support for thread local configuration when run  cross browser tests in parallel

### ISSUE THREAD

**Issue by softommy** [selenide/selenide#1385]
![image](https://user-images.githubusercontent.com/35088786/105846491-38e55500-6017-11eb-81f0-1c7a96fbbbf3.png)

**Comment by asolntsev** [comment]
Duplicate of https://github.com/selenide/selenide/issues/1372
